.github/workflows/merge_branch.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync master and next-moc
+on:
+  push:
+    branches:
+      - master
+jobs:
+  sync-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Merge master -> next-moc
+        uses: robotology/gh-action-nightly-merge@v1.2.0
+        with:
+          stable_branch: 'master'
+          development_branch: 'next-moc'
+          allow_ff: false
```

Comment:
We could even allow that; this way we don’t get pointless merge commits in those times when `next-moc` and `master` are actually the same.

Reply:
Done